attestation-service/docs/policy.md

```diff
@@ -37,6 +37,14 @@ If the extension is registered in the EAR specification, the key will be positiv
 Otherwise, the key should be negative.
 The extension value can be any valid JSON, including nested objects.
 
+The default policy includes the extension `ear.trustee.identifiers`, which is used
+to store fields that identify the workload.
+The `identifiers` extension includes two subfields.
+
+* `validated` identifiers are those that are bound to the hardware evidence via
+attestation and that the guest runtime binds to the workload.
+* `unvalidated` identifiers are anything else that describe the workload.
```

Comment:
Any examples on how do you see the validated identifiers being used?

Reply:
This is still in discussion, but the basic idea is that the KBS (or some other consumer of the attestation token) will use the identifiers plus some policy to come up with one top-level identity for the pod, such as a SPIFFE ID. For instance, you could use selectors over the identifiers.

I plan to add some more information about this and about the details of each identifier as things evolve.